WPMedia/fusion-news-theme-blocks
# Updated Promo Block External Images

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Update all 8 promo blocks to use the signing service content source to resize external images, or images from PC that do not already have an auth value

## Jira Ticket

- [THEMES-1108](https://arcpublishing.atlassian.net/browse/THEMES-1108)

## Acceptance Criteria

For all 8 promo blocks where the image url is configurable: 

manual - xl, l, m, s promo blocks
regular - xl, l, m, s promo blocks

The block should correctly resize/render images from external URLs when the URL is entered in the custom fields.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-1108`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/large-manual-promo-block,@wpmedia/extra-large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/small-manual-promo-block`
3. For each manual promo block, make sure that the block renders images from both Photocenter and external URLs (here's a test image you can use: https://testimages.org/img/testimages_screenshot.jpg)
4. For each promo block, make sure that the block renders images from global content, Photocenter, and external URLs

## Effect Of Changes

### Before

Promo Blocks would not work with external URLs
<img width="792" alt="image" src="https://github.com/WPMedia/arc-themes-blocks/assets/85515364/9e410bce-6db4-42b5-b35b-a73d3d6803bb">

### After

External image URLs are resized and render correctly in the promo blocks


## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1108]: https://arcpublishing.atlassian.net/browse/THEMES-1108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
